src/wp-includes/author-template.php

```diff
@@ -229,13 +229,29 @@ function the_author_meta( $field = '', $user_id = false ) {
  */
 function get_the_author_link() {
 	if ( get_the_author_meta( 'url' ) ) {
-		return sprintf(
+		global $authordata;
+
+		$author_url          = get_the_author_meta( 'url' );
+		$author_display_name = get_the_author();
+
+		$author_url_link_html = sprintf(
```

Comment:
Looking at the related functions, I think just using the `$link` name here would be more consistent with similar code elsewhere. It's also the name passed to `the_author_posts_link` and `author_link` filters. I don't necessarily object to a more specific name, just prefer consistency in naming when looking at the bigger picture :)